share/completions/loadkeys.fish

```diff
@@ -0,0 +1,15 @@
+complete -fc loadkeys -a "(find /usr/share/kbd/keymaps/ -type f | string replace -rf '.*/(.*).map.gz' '\$1')"
```

Comment:
I'm not sure these *need* to be .gz, so I'd probably make that second extension optional, maybe even up to

```fish
string replace -rf '.*/(.*).map(\..*)' '\$1'
```

Reply:
You're right, and I also just found out that `/usr/share/kbd/keymaps` isn't as standardized as the manual I read made it seem. I will update this later.

Reply:
I've updated the PR to work with all paths I could find for different distributions, and all extensions that `loadkeys` looks for, according to its source code.